libraries/cmake/source/modules/utils.cmake

```diff
@@ -0,0 +1,85 @@
+# Copyright (c) 2014-present, Facebook, Inc.
+# All rights reserved.
+#
+# This source code is licensed in accordance with the terms specified in
+# the LICENSE file found in the root directory of this source tree.
+
+cmake_minimum_required(VERSION 3.13.3)
+
+option(OSQUERY_THIRD_PARTY_SOURCE_MODULE_WARNINGS "This option can be enable to show all warnings in the source modules. Not recommended" OFF)
+
+function(getGitExecutableName output_variable)
```

Comment:
We have to replace this with `find_package(git)`

Reply:
Done together with the patch system, will push soon.